Tests/ViewInspectorTests/ViewModifiers/EnvironmentModifiersTests.swift

```diff
@@ -17,6 +17,15 @@ final class ViewEnvironmentTests: XCTestCase {
         let sut2 = EmptyView().environment(\.colorScheme, .light)
         XCTAssertEqual(try sut2.inspect().emptyView().environment(\.colorScheme), .light)
     }
+
+    func testClosureEnvironmentValue() {
+        let value: (_ value: String) -> Bool = { value in value == "hello world" }
+        let sut = EmptyView().environment(\.testHandlerClosure, value)
+        XCTAssertNoThrow(try sut.inspect().emptyView())
+        XCTAssertNoThrow(try sut.inspect().emptyView().environment(\.testHandlerClosure))
```

Comment:
Before my change this line failed with:
> testClosureEnvironmentValue: XCTAssertNoThrow failed: threw error "emptyView() found ModifiedContent<SwiftUI.EmptyView, SwiftUI._EnvironmentKeyWritingModifier<(Swift.String) -> Swift.Bool>> instead of EmptyView"